webapp/components/admin_console/legal_and_support_settings.jsx

```diff
@@ -130,7 +130,7 @@ export default class LegalAndSupportSettings extends AdminSettings {
                     helpText={
                         <FormattedMessage
                             id='admin.support.problemDesc'
-                            defaultMessage='Link to help documentation from team site main menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues.'
+                            defaultMessage='Link to help documentation from the Main Menu. By default this points to the peer-to-peer troubleshooting forum where users can search for, find and request help with technical issues. Leave this field blank to hide the option from the Main Menu.'
```

Comment:
The URL for the Report a Problem link in the Main Menu. If this field is empty, the link is removed from the Main Menu.